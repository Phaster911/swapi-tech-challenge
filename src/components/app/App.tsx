import { COLORS } from "../../style-constants";
import styled from "styled-components";
import { QueryClientProvider } from "@tanstack/react-query";
import createQueryClient from "../../react-query/create-queryclient";
import Home from "../home/Home";

const AppWrapper = styled.div`
  height: 100dvh;
  display: flex;
  flex-direction: column;
  color: ${COLORS.text};
`;

export default function App() {
  return (
    <AppWrapper>
      <QueryClientProvider client={createQueryClient()}>
        <Home />
      </QueryClientProvider>
    </AppWrapper>
  );
}
