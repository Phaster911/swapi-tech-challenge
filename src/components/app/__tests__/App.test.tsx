import { render } from "@testing-library/react";
import App from "../App";

describe("<App />", () => {
  it("renders the component", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
