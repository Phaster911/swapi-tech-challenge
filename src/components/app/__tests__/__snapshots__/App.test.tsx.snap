// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<App /> > renders the component 1`] = `
.c0 {
  height: 100dvh;
  display: flex;
  flex-direction: column;
  color: #111111;
}

<div>
  <div
    class="c0"
  >
    <div>
      Home
    </div>
  </div>
</div>
`;
